doc/examples/segmentation/plot_rolling_ball.py

```diff
@@ -0,0 +1,144 @@
+"""
+=======================
```

Comment:
You need 52 equal signs here, for the line to be the same length as the current title (line 3).

Reply:
fixed